src/inc/msquic.h

```diff
@@ -118,6 +118,7 @@ typedef enum QUIC_CREDENTIAL_FLAGS {
     QUIC_CREDENTIAL_FLAG_USE_SYSTEM_MAPPER                      = 0x00010000, // Schannel only
     QUIC_CREDENTIAL_FLAG_CACHE_ONLY_URL_RETRIEVAL               = 0x00020000, // Windows only currently
     QUIC_CREDENTIAL_FLAG_REVOCATION_CHECK_CACHE_ONLY            = 0x00040000, // Windows only currently
+    QUIC_CREDENTIAL_FLAG_SCHANNEL_SERIALIZED_PEER_CERT          = 0x00080000, // Schanel only
```

Comment:
I don't care for having `SCHANNEL` in the name. We already have plenty of Schannel specific flags that we just comment as such. Let's generalize this a bit more. Who knows? Perhaps we add support to OpenSSL one day for some reason...

Reply:
call this `IN_PROC_PEER_CERTIFICATE`

Reply:
fix typo in comment